contracts/cw3-flex-multisig/src/msg.rs

```diff
@@ -21,55 +21,22 @@ pub struct InitMsg {
 /// individual voter used in tallying should be snapshotted at the beginning of
 /// the block at which the proposal starts (this is likely the responsibility of a
 /// correct cw4 implementation).
+/// See also `ThresholdResponse` in the cw3 spec.
 #[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
 #[serde(rename_all = "snake_case")]
 pub enum Threshold {
-    /// Declares that a fixed weight of yes votes is needed to pass.
-    /// It does not matter how many no votes are cast, or how many do not vote,
-    /// as long as `weight` yes votes are cast.
-    ///
-    /// This is the simplest format and usually suitable for small multisigs of trusted parties,
-    /// like 3 of 5. (weight: 3, total_weight: 5)
-    ///
-    /// A proposal of this type can pass early as soon as the needed weight of yes votes has been cast.
+    /// Declares that a fixed weight of Yes votes is needed to pass.
+    /// See `ThresholdResponse.AbsoluteCount` in the cw3 spec for details.
     AbsoluteCount { weight: u64 },
 
-    /// Declares a percentage of the total weight that must cast yes votes in order for
-    /// a proposal to pass. The percentage is computed over the total weight minus the weight of the
-    /// abstained votes.
-    ///
-    /// This is useful for similar circumstances as `AbsoluteCount`, where we have a relatively
-    /// small set of voters and participation is required. The advantage here is that if the
-    /// voting set (group) changes between proposals, the number of votes needed is adjusted
-    /// accordingly.
-    ///
-    /// Example: we set `percentage` to 51%. Proposal 1 starts when there is a `total_weight` of 5.
-    /// This will require 3 weight of yes votes in order to pass. Later, the Proposal 2 starts but the
-    /// `total_weight` of the group has increased to 9. That proposal will then automatically
-    /// require 5 yes of 9 to pass, rather than 3 yes of 9 as would be the case with `AbsoluteCount`.
-    ///
-    /// A proposal of this type can pass early as soon as the needed weight of yes votes has been cast.
+    /// Declares a percentage of the total weight that must cast Yes votes in order for
+    /// a proposal to pass.
+    /// See `ThresholdResponse.AbsolutePercentage` in the cw3 spec for details.
```

Comment:
Yes, they are possible, and are used extensively in the crates documentation. Here in particular, I first added and afterwards then removed the hyperlink formatting, because these comments go to the json files.

We definitely should start using these links, and confirm that they are working correctly in the crates docs.

Reply:
> (Side-note, we really need to review the RustDoc output once we push 0.4.0)

I propose we create an issue in cosmwasm for that, and consider / mention all the projects.

Reply:
Oh, there are issues... for quite some time:

https://github.com/CosmWasm/cosmwasm/issues/436

https://github.com/CosmWasm/cosmwasm/issues/196

Just seems that all of us prefer to code than tidy up RustDoc